gtk2/gtk2.json

```diff
@@ -90,12 +90,12 @@
         "--disable-introspection",
         "--disable-python2"
       ],
-      "ensure-writable": [
-        "/lib/gtk-2.0/2.10.0/immodules.cache"
+      "build-commands": [
+        "make -C client/gtk2 install"
       ],
       "post-install": [
-        "install -m644 --target-directory=${FLATPAK_DEST}/lib/gtk-2.0/2.10.0/immodules client/gtk2/.libs/im-ibus.so",
-        "gtk-query-immodules-2.0 > ${FLATPAK_DEST}/lib/gtk-2.0/2.10.0/immodules.cache"
+        "gtk-query-immodules-2.0 > immodules.cache",
+        "install immodules.cache -t $(pkg-config --variable=libdir gtk+-2.0)/gtk-2.0/$(pkg-config --variable=gtk_binary_version gtk+-2.0)/"
```

Comment:
I realize its more accurate, but gtk2 also hasn't had an ABI break since like 2008, I don't think we need to query the binary version.

Reply:
It's a directory path. I.e. it normally becomes `/app/lib/gtk-2.0/2.10.0/`.

Reply:
Yes, my point was `2.10.0` is a string that will never change to the end of time, bit of a complication for no gain.

Reply:
Oh, I see.